doc/classes/File.xml

```diff
@@ -320,7 +341,32 @@
 				    var read2 = f.get_16() # 121
 				    var converted1 = unsigned16_to_signed(read1) # -42
 				    var converted2 = unsigned16_to_signed(read2) # 121
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				public const int Max15B = 1 &lt;&lt; 15;
+				public const int Max16B = 1 &lt;&lt; 16;
+
+				public short Unsigned16ToSigned(ushort unsigned)
+				{
+				    return unchecked((short)((unsigned + Max15B) % Max16B - Max15B));
+				}
+
+				public override void _Ready()
+				{
+				    var f = new File();
+				    f.Open("user://file.dat", File.ModeFlags.WriteRead);
+				    f.Store16(unchecked((ushort)-42)); // This wraps around and stores 65494 (2^16 - 42).
+				    f.Store16(121); // In bounds, will store 121.
+				    f.Seek(0); // Go back to start to read the stored value.
+				    ushort read1 = f.Get16(); // 65494
+				    ushort read2 = f.Get16(); // 121
+				    short converted1 = Unsigned16ToSigned(read1); // -42
+				    short converted3 = BitConverter.ToInt16(BitConverter.GetBytes(read1), 0); // -42
+				    short converted2 = Unsigned16ToSigned(read2); // 121
+				    short converted4 = BitConverter.ToInt16(BitConverter.GetBytes(read2), 0); // 121
+				}
```

Comment:
Since there is a more "native" way to do this in C#, I don't think we need the `Unsigned16ToSigned` method which is needed as a workaround in GDScript.

```suggestion
				public override void _Ready()
				{
				    var f = new File();
				    f.Open("user://file.dat", File.ModeFlags.WriteRead);
				    f.Store16(unchecked((ushort)-42)); // This wraps around and stores 65494 (2^16 - 42).
				    f.Store16(121); // In bounds, will store 121.
				    f.Seek(0); // Go back to start to read the stored value.
				    ushort read1 = f.Get16(); // 65494
				    ushort read2 = f.Get16(); // 121
				    short converted1 = BitConverter.ToInt16(BitConverter.GetBytes(read1), 0); // -42
				    short converted2 = BitConverter.ToInt16(BitConverter.GetBytes(read2), 0); // 121
				}
```